snapshot/pkg/volume/openebs/processor.go

```diff
@@ -29,14 +29,23 @@ import (
 	crdv1 "github.com/kubernetes-incubator/external-storage/snapshot/pkg/apis/crd/v1"
 	"github.com/kubernetes-incubator/external-storage/snapshot/pkg/cloudprovider"
 	"github.com/kubernetes-incubator/external-storage/snapshot/pkg/volume"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	openEBSPersistentDiskPluginName = "openebs"
 )
 
+var (
+	// supportedCasType is a map of supported volume type for snapshot
+	// operations.
+	supportedCasType = map[string]bool{
```

Comment:
name doesn't reflect what it is intended.. what about `snapSupportedCASTypes`
and why 'bool'?

Reply:
it can be plain `[]string`

Reply:
@vishnuitta the effective way has been mentioned here https://golang.org/doc/effective_go.html#maps